desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -751,7 +751,7 @@ protected boolean onWindowClosing() {
 
                 if (unsavedChangesChoice == 0) { // save changes
                     _saveButton.doClick();
-                } else if (unsavedChangesChoice == 2) { // cancel closing
+                } else if (unsavedChangesChoice != 1) { // cancel closing
```

Comment:
Oh I thought this was trivial :-) Ehm, then I suggest we just merge as-is since there's at least no obvious better choice!